db/schema.rb

```diff
@@ -14,16 +14,16 @@
 
   create_table "absences", force: :cascade do |t|
     t.integer  "member_id"
-    t.integer  "performanceset_id"
```

Comment:
Looking over the changes and the schemas on prod and staging, it looks like prod matches the old schema.rb file, not the migrations. I'm going to make a couple changes to the prod database, which I will mention here. 

I'm going to make the following changes:
- On `absences`, rename `performanceset_id` to `performance_set_id` and the index. This will support the change you made in #4, which would otherwise break production

- On `member_sets`, rename `set_id` to `performance_set_id` and the index. This will support the change you made in #7, which would otherwise break production

The change to `set_member_instruments`, adding `performance_set_id`, is not desirable, and I'll make a migration to roll that back.

After that, the schema on staging and prod should be equal to what you get from migrations except for a couple scattered unused indicies that I'm not going to stress at the moment.

Thanks so much, this is definitely moving in the right direction.